dotnet/src/Experimental/Process.Abstractions/KernelProcessEdge.cs

```diff
@@ -1,31 +1,37 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
+using System.Runtime.Serialization;
+
 namespace Microsoft.SemanticKernel;
 
 /// <summary>
 /// A serializable representation of an edge between a source <see cref="KernelProcessStep"/> and a <see cref="KernelProcessFunctionTarget"/>.
 /// </summary>
+[DataContract]
```

Comment:
Curious, is dapr using binary serializer?  Haven't seen data-contracts in a while.  Would this impact the serialization pattern @esttenorio is exploring?  (are we bound to two serialziation schemes, or one?).  In anycase, it might not be a bad idea to round-trip serialization in some unit-tests.

Reply:
Currently Dapr uses Json for state serialization but uses DataContract for remoting actor method calls, they are working on updating this to json. So unfortunately, we need to support multiple serialization methods on some of these contracts.